ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```
@@ -502,6 +502,19 @@ tests:
       DISCONNECTED: "true"
       IP_STACK: v6
     workflow: assisted-baremetal-operator-ztp
+- as: e2e-ai-operator-ztp-3masters-day2-workers
```

Consider using more specific terminology

```suggestion
- as: e2e-ai-operator-ztp-compact-day2-workers
```

---

Right, lets use `compact/SNO` vs `3masters/1master`